ci/unit.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 
```

Comment:
tag value should be `1.13.8`. I'm surprised this didn't auto generate.

Reply:
👍